playground/ci/verify_beam_example.py

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO implement verification of example
+def verify_example(code, sdk):
+    """Verify example by his code and SDK
+
+    Send request to the backend on AppEngine and receive output of code running.
+
+    Args:
+        code: code of the example.
+        sdk: SDK of the example.
+
+    Returns:
+        In case of some request returns error - log error and returns None.
+        In case of backend successfully returns an output - return output.
+    """
+    return ""
```

Comment:
Could you please add a new line at the end of this file?

Reply:
Added.